### PR TITLE
Create new mail message

### PR DESCRIPTION
**Pull request by KmlBaranowski**
Create new mail in messaging system

### CODE REVIEW

File: integration_test/create_mail_test.dart [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import 'src/common_methods.dart';
+import 'package:patrol/patrol.dart';
+import 'package:gallery/keys.dart';
+
+void main() {
+  patrolTest('Create new mail', nativeAutomation: true, ($) async {
+    await startAppTest($);
+    await $(K.studyDemoList).tap();
+    await $(K.mailList).waitUntilVisible();
```
Comment (jBorkowska) [inline integration_test/create_mail_test.dart]: Is this necessary?

File: lib/studies/reply/adaptive_nav.dart [hunk 2]
```diff
@@ -1219,7 +1220,7 @@ class _ReplyFabState extends State<_ReplyFab>
               return Tooltip(
                 message: tooltip,
                 child: InkWell(
-                  key: const ValueKey('ReplyFab'),
+                  key: K.createMailButton,
```
Comment (jBorkowska) [inline lib/studies/reply/adaptive_nav.dart]: Let's preserve values of the keys that was here before, like you did with `studyDemoList`.

File: lib/studies/reply/mail_card_preview.dart [hunk 3]
```diff
@@ -59,7 +60,7 @@ class MailPreviewCard extends StatelessWidget {
           return mailPreview;
         } else {
           return Dismissible(
-            key: ObjectKey(email),
+            key: K.mailList,
```
Comment (jBorkowska) [inline lib/studies/reply/mail_card_preview.dart]: As in previous comment

**Review by jBorkowska — CHANGES_REQUESTED** [review]
Also check warnings and correct those that are in your code

File: integration_test/create_mail_test.dart (hunk 1 above)
Comment (KmlBaranowski) [reply integration_test/create_mail_test.dart]: Yes, i moved the lines to common_methods.dart and the changes can be viewed in the upcoming commit

Comment (jBorkowska) [reply integration_test/create_mail_test.dart]: I still don't understand why you call `await $(K.mailList).waitUntilVisible();`

Comment (jBorkowska) [reply integration_test/create_mail_test.dart]: What it checks? Is it check for being in inbox?

Comment (KmlBaranowski) [reply integration_test/create_mail_test.dart]: Yes this checks if the bot has entered mailbox demonstration

File: integration_test/star_mail_test.dart
Comment (jBorkowska) [inline integration_test/star_mail_test.dart:1]: I thought it shouldn't be in this PR, but if you did place it here then let's leave it

**Review by jBorkowska — APPROVED** [review]

**Pull request merged**